### PR TITLE
docs: revise the bastion ssh guideline to mitigate permission denied errors

### PR DESCRIPTION
**Pull request by techieforfun**
- Updated the username variable to avoid encountering the "Permission denied (publickey)" error in different AMIs
- Removed redundant description.

### CODE REVIEW

**Comment by justinsb** [comment]
Thanks @techieforfun - these changes lgtm!

/ok-to-test
/lgtm
/approve

/hold for a few hours to give others a chance to review, but lgtm

File: docs/bastion.md [hunk 1]
```diff
@@ -162,10 +138,11 @@ ssh-add -l
 # If you need to add the key to your agent:
 ssh-add path/to/private/key
 
-# Now you can SSH into the bastion
-ssh -A admin@<bastion-ELB-address>
+# Now you can SSH into the bastion, where <username> of Ubuntu in AWS is `ubuntu` and of Debian is `admin`, <bastion-ELB-address> is the URL of the assigned Load Balancer.
+ssh -A <username>@<bastion-ELB-address>
```
Comment (johngmyers) [inline docs/bastion.md:142]:
```suggestion
# Now you can SSH into the bastion. Substitute the administrative username of the instance's OS for <username> (`ubuntu` for  Ubuntu,  `admin` for Debian, etc.) and the bastion domain for <bastion-domain>. If the bastion doesn't have a public CNAME alias, use the domain of the assigned load balancer as the bastion domain.
ssh -A <username>@<bastion-domain>
```

File: docs/bastion.md [hunk 2]
```diff
@@ -162,10 +138,11 @@ ssh-add -l
...
 
-# Where <bastion-ELB-address> is usually bastion.$clustername (bastion.example.kubernetes.cluster) unless otherwise specified
+# then you can SSH into control planes or worker nodes in the cluster, using the forwarded auth key
```
Comment (johngmyers) [inline docs/bastion.md:144]:
```suggestion
# then you can use the fowarded authentication to SSH into control-plane or worker nodes in the cluster.
```

**Review by johngmyers — CHANGES_REQUESTED** [review]
The "Access when using gossip" section also applies to none-dns. This isn't a blocking comment.

**Comment by hakman** [comment]
@techieforfun Could you also squash the commits? Thanks!

**Comment by techieforfun** [comment]
it's ready @hakman

**Comment by hakman** [comment]
Thanks @techieforfun!
/lgtm
/hold cancel

**Pull request merged**